scripts/asciidoctor-text/convert-it-all.py

```diff
@@ -0,0 +1,86 @@
+"""Utility script to convert OCP docs from adoc to plain text."""
```

Comment:
How did we derive this conversion logic? Is it our own experimentation or the same way that docs team has converted our docs to plain text format initially?

Reply:
Someone from the docs team helped us out, this is from https://gitlab.cee.redhat.com/dmacpher/asciidoctor-text.git